editor/store/selectors.js

```diff
@@ -76,7 +76,7 @@ export function hasEditorRedo( state ) {
  * @return {boolean} Whether the post is new.
  */
 export function isEditedPostNew( state ) {
-	return getCurrentPost( state ).status === 'auto-draft';
+	return getEditedPostAttribute( state, 'status' ) === 'auto-draft';
```

Comment:
I'm not certain about this change. Because `getEditedPostAttribute` checks the edited values as well. So if someone sets the status to "draft" or something else and diidn't save yet. There will be a difference. The post won't be considered new while it's still new.